docs/setup/sso.md

```diff
@@ -385,3 +385,68 @@ upstream_oauth2:
           template: "{{ user.email }}"
           set_email_verification: always
 ```
+
+
+### Rauthy
+
+1. Click `Clients` in the Rauthy Admin sidebar and click `Add new Client`
+2. Fill in the fields as below:
+
+   | Field | Value |
+   |-----------|-----------|
+   | Client ID | `matrix-authentication-service` |
+   | Client Name | `matrix-authentication-service` |
+   | Redirect URI | `https://<auth-service-domain>/upstream/callback/<id>` |
+
+3. Set the client to be `Confidential`.
+
+4. Click `Save`
+
+5. Select the client you just created from the clients list.
+6. Enable the `authorization_code`, and `refresh_token` grant types.
+7. Set the allowed scopes to `openid`, `profile`, and `email`.
+8. Set both Access Algorithm and ID Algorithm to `RS256`.
+9. Set PKCE challenge method to `S256`.
+10. Click `Save`
+11. Copy the `Client ID` from the `Config` tab and the `Client Secret` from the `Secret` tab.
+
+
+Authentication service configuration:
+
+```yaml
+upstream_oauth2:
+  providers:
+    - id: "01JFFHK7HJF70YSYF753GEWVRP"
+      human_name: Rauthy
+      brand_name: "rauthy"
```

Comment:
The 'brand name's are for some well-known brands which have logos in various places. The 'human name' is the one that really matters, as it will be displayed to end users.


```suggestion
```

Example on this screenshot (one with "brand_name: github + human_name: GitHub", the other with no "brand_name" and "human_name: Company SSO"):
<img width="502" alt="image" src="https://github.com/user-attachments/assets/4c5fd539-5208-413b-85bc-f4b5fbe8e128" />